app/views/products/_form.html.erb

```diff
@@ -0,0 +1,7 @@
+<%= form_for @product do |form| %>
+  <%= form.text_field :name %>
+  <%= form.text_field :price %>
+  <%= form.text_field :short_description %>
+
+  <%= form.submit t(".submit_product") %>
+<% end %>
```

Comment:
@desi Yeah I was going to, but simple didn't know what to do with the price field because it's reading it as money type. I forgot that I can just set it to a text field, so I did that and it's a simple_form form now.